src/CLIFramework/IO/NullStty.php

```diff
@@ -0,0 +1,23 @@
+<?php
+namespace CLIFramework\IO;
+
+class NullStty implements Stty
```

Comment:
Ummm, I feel like that directly using `UnixStty` is enough. We can ignore the return value of `shell_exec`, `system` or `exec` if they are not available.